### PR TITLE
Fix/iasystem

### PR DESCRIPTION
**Pull request by fanglinfang**

### CODE REVIEW

File: rc_django/views.py [hunk 1]
```diff
@@ -78,9 +75,9 @@ def proxy(request, service, url):
 
     start = time()
     try:
-        if service == "iasystem" and subdomain is not None:
-            response = dao.getURL(url, headers, subdomain)
-
+        if re.match(r'^iasystem_', service):
```
Comment (jlaney) [inline rc_django/views.py]: I think you should get rid of this block, and just set the headers for iasystem on line 51.

File: rc_django/views.py [hunk 2]
```diff
@@ -78,9 +75,9 @@ def proxy(request, service, url):
...
+            headers = {"Accept": "application/vnd.collection+json"}
```
Comment (jlaney) [inline rc_django/views.py]: Just in case other headers have already been set, you should use:

headers["Accept"] = "application/vnd.collection+json"

Comment (fanglinfang) [reply rc_django/views.py]: Thank you for pointing that out!

File: rc_django/views.py [hunk 3]
```diff
@@ -46,24 +47,20 @@ def proxy(request, service, url):
     use_pre = False
     headers = {}
 
+    if service == "iasystem":
+        if url.endswith('/evaluation'):
```
Comment (jlaney) [inline rc_django/views.py]: Set the header here?
headers["Accept"] = "application/vnd.collection+json"

Comment (fanglinfang) [reply rc_django/views.py]: Ah, added it back. Thank you!

**Pull request merged**